templates/default/images/icon_default.svg

```diff
@@ -1,144 +1,473 @@
+		C27.5,26.2,27.1,26.7,26.5,26.7z"/>
+	<path class="st5" d="M25.3,26.7H6.7c-0.6,0-1-0.4-1-1V7.1c0-0.6,0.4-1,1-1h18.6c0.6,0,1,0.4,1,1v18.6
+		C26.3,26.2,25.8,26.7,25.3,26.7z"/>
+	<g id="abgerundet" class="st6">
+		<path class="st7" d="M11,24H7V7h4V4H6.7C5.3,4,4,4.8,4,6.1v18C4,25.5,5.3,27,6.7,27H11V24z"/>
+		<path class="st7" d="M25.3,4H21v3h4v17h-4v3h4.3c1.4,0,2.7-1.5,2.7-2.8v-18C28,4.8,26.7,4,25.3,4z"/>
+	</g>
+	<g class="st6">
+		<polygon class="st7" points="10,25 7,25 7,8 10,8 10,5 4,5 4,28 10,28 		"/>
+		<polygon class="st7" points="22,5 22,8 25,8 25,25 22,25 22,28 28,28 28,5 		"/>
+	</g>
+	<polygon class="st8" points="6,26 6,7 10,7 10,5 4,5 4,28 10,28 10,26 	"/>
+	<g>
+		<polygon class="st9" points="10,25 6,25 6,7 10,7 10,5 4,5 4,27 10,27 		"/>
+		<polygon class="st9" points="22,5 22,7 26,7 26,25 22,25 22,27 28,27 28,5 		"/>
+	</g>
+</g>
+<g id="tst">
+</g>
+<g id="file">
+</g>
+<g id="usr">
+</g>
+<g id="webr">
+</g>
+<g id="svy">
+</g>
+<g id="sahs">
+</g>
+<g id="exc">
+</g>
+<g id="grp">
+</g>
+<g id="fold">
+</g>
+<g id="ok">
+</g>
+<g id="not_x5F_ok">
+</g>
+<g id="accs">
+</g>
+<g id="adm">
+</g>
+<g id="adve">
+</g>
+<g id="alert">
+</g>
+<g id="ass">
+</g>
+<g id="auth">
+</g>
+<g id="awra">
+</g>
+<g id="bibl">
+</g>
+<g id="bibs">
+</g>
+<g id="blga">
+</g>
+<g id="blog">
+</g>
+<g id="bm">
+</g>
+<g id="bmf">
+</g>
+<g id="book_1_" class="st6">
+</g>
+<g id="cadm">
+</g>
+<g id="cal">
+</g>
+<g id="cal2">
+</g>
+<g id="calg">
+</g>
+<g id="cals">
+</g>
+<g id="chap">
+</g>
+<g id="crss">
+</g>
+<g id="exc_1_">
+</g>
+<g id="extt">
+</g>
+<g id="frm">
+</g>
+<g id="frma">
+</g>
+<g id="glo">
+</g>
+<g id="grp_1_">
+</g>
+<g id="grpr">
+</g>
+<g id="grps">
+</g>
+<g id="hlps">
+</g>
+<g id="htlm">
+</g>
+<g id="inbox">
+</g>
+<g id="info">
+</g>
+<g id="itgr">
+</g>
+<g id="lm">
+</g>
+<g id="lng">
+</g>
+<g id="lngf">
+</g>
+<g id="lobj">
+</g>
+<g id="local">
+</g>
+<g id="logs">
+</g>
+<g id="lres">
+</g>
+<g id="lrss">
+</g>
+<g id="mail">
+</g>
+<g id="mcst">
+</g>
+<g id="mcts">
+</g>
+<g id="mds">
+</g>
+<g id="mep">
+</g>
+<g id="mob">
+</g>
+<g id="mobs">
+</g>
+<g id="mostly_x5F_ok">
+</g>
+<g id="ms">
+</g>
+<g id="nwss">
+</g>
+<g id="obj">
+</g>
+<g id="orgu">
+</g>
+<g id="otpl">
+</g>
+<g id="page">
+</g>
+<g id="pays">
+</g>
+<g id="pd">
+</g>
+<g id="pdts">
+</g>
+<g id="pg">
+</g>
+<g id="pg_x5F_d">
+</g>
+<g id="pg_x5F_d_x5F_sc">
+</g>
+<g id="pg_x5F_del">
+</g>
+<g id="pg_x5F_del_x5F_sc">
+</g>
+<g id="pg_x5F_sc">
+</g>
+<g id="pin">
+</g>
+<g id="file_x5F_inline">
+</g>
+<g id="feed">
+</g>
+<g id="facs">
+</g>
+<g id="excv">
+</g>
+<g id="excs">
+</g>
+<g id="ecss">
+</g>
+<g id="drafts">
+</g>
+<g id="crsr">
+</g>
+<g id="dcl_x5F_fold">
+</g>
+<g id="dcl_x5F_file">
+</g>
+<g id="dcl">
+</g>
+<g id="crsv">
+</g>
+<g id="crst">
+</g>
+<g id="cmps">
+</g>
+<g id="cld">
+</g>
+<g id="chtr">
+</g>
+<g id="chta">
+</g>
+<g id="cat_1_">
+</g>
+<g id="cert">
+</g>
+<g id="catr_1_">
+</g>

```

Comment:
As far as I understand SVG, we do not need these groups, they rather seem to be artifacts from CKs editor.